docs/cli.md

```diff
@@ -4,98 +4,155 @@
+  ```
+  ### 1. enable
+  You can temparaily enable or disable the plugin.
+  ```
+  ~$ ./vatz plugin enable <plugin_id> <true/false>
+  ```
+  ### 2. install
+  
+  First, you can install the plugin by CLI. For installing the plugin, you have to know the repository URL where the plugin is implemented.
+  For example, 
+  ```
+  ~$ ./vatz plugin install github.com/dsrvlabs/vatz-plugin-cosmoshub/plugins/node_active_status cosmos-status
+  ```
+
+  The last argument, `cosmos-status` is a simple name that is used for binary name on your machine. So, you could set the plugin name as desired.
+  ### 3. list
+  If you install some plugin, you can query installed plugins by list subcommand.
+
+  ```
+  ~$ ./vatz plugin list
+  ```
+  ```
+  2023-05-26T14:40:10+09:00 INF Load Config default.yaml module=config
+  2023-05-26T14:40:10+09:00 INF List plugins module=plugin
+  2023-05-26T14:40:10+09:00 INF Create DB Instance module=db
+  2023-05-26T14:40:10+09:00 INF List module=plugin
+  2023-05-26T14:40:10+09:00 INF List module=db
+  +---------------+------------+---------------------+----------------------------------------------------------------------+---------+
+  | NAME          | IS ENABLED | INSTALL DATE        | REPOSITORY                                                           | VERSION |
+  +---------------+------------+---------------------+----------------------------------------------------------------------+---------+
+  | cosmos-status | true       | 2023-05-26 14:40:00 | github.com/dsrvlabs/vatz-plugin-cosmoshub/plugins/node_active_status | latest  |
+  +---------------+------------+---------------------+----------------------------------------------------------------------+---------+
+  ```
+  ### 4. start
+  There are 4 flags under `plugin start`
+
+  ```
+  ~$ ./vatz plugin start -h
+  Start installed plugin
+
+  Usage:
+    plugin start [flags]
+
+  Examples:
+  vatz plugin start pluginName
+
+  Flags:
+    -a, --args string     Arguments
+    -h, --help            help for start
+    -l, --log string      Logfile
+    -p, --plugin string   Installed plugin name
+  ```
+
+  You can start installed plugin like below.
+  ```
+  ~$ ./vatz plugin start --plugin <pluginName>
+  ```
+  For some plugins that require arguments for binary, you can provide it with `--args` flag. It should be surrounded by quotes like as one string. For example,
+
+  ```
+  ~$ ./vatz plugin start --plugin cosmos-status --args "--valoperAddr=<YOUR VALOPER ADDRESS>" 
+  ```
+
+  ```
+  2022-11-21T02:57:14Z INF Start plugin cosmos-status%!(EXTRA string=--valoperAddr=<HIDDEN>) module=plugin
+  2022-11-21T02:57:14Z INF Start plugin cosmos-status module=plugin
+  2022-11-21T02:57:14Z INF newReader module=db
+  2022-11-21T02:57:14Z INF Create DB Instance module=db
+  ```
+
+### 5. status
+When VATZ is running, you can check status of plugins(OK or FAIL).
+
+  ```
+  ~$  ./vatz plugin status
+  ```
+
+### 6. stop
```

Comment:
@qkrybin 
I think example is kinda wrong here. 
Can you check?

Reply:
fixed!

Reply:
Confirmed!!

<img width="1115" alt="image" src="https://github.com/dsrvlabs/vatz/assets/6308023/371a7686-f712-47fe-b967-eaf690f736b7">